docs/migrate-to-antrea.md

```diff
@@ -0,0 +1,147 @@
+# Migrate from other CNI to Antrea
+
+This document describes how to migrate from other CNIs to Antrea. Currently, we support migration
+from Calico and Flannel.
+As for Calico, NetworkPolicy conversion is supported conditionally. As for Flannel, only Pod
+networking migration is supported.
+
+NOTE: The migration process is only verified on Kubernetes v1.27.3, Calico v3.26 and Flannel
+v0.22.3. Other versions may not be supported. And the migration process only supports
```

Comment:
Since the solution is generic, I feel the guide doesn't need to be specific to CNIs. Like
"The following is a list of CNIs and versions we have verified the migration process, for your reference. CNIs and versions that are not listed probably work, please [create an issue](https://...) if you run into problems when migrating to Antrea."

Reply:
Done.